Source/Csla/Server/DataPortal.cs

```diff
@@ -29,63 +30,44 @@ public class DataPortal : IDataPortalServer
     /// <summary>
     /// Gets the data portal dashboard instance.
     /// </summary>
-    public static Dashboard.IDashboard Dashboard { get; internal set; }
+    public static IDashboard Dashboard { get; internal set; }
+
+    private DataPortalServerOptions Options { get; set; }
+    private IAuthorizeDataPortal Authorizer { get; set; }
+    private List<IInterceptDataPortal> InterceptorList { get; set; }
+    private IObjectFactoryLoader FactoryLoader { get; set; }
+    private IDataPortalActivator Activator { get; set; }
+    private IDataPortalExceptionInspector ExceptionInspector { get; set; }
 
     /// <summary>
     /// Creates an instance of the type.
     /// </summary>
     /// <param name="applicationContext"></param>
     /// <param name="dashboard"></param>
-    public DataPortal(ApplicationContext applicationContext, IDashboard dashboard)
+    /// <param name="options"></param>
+    /// <param name="activator"></param>
+    /// <param name="authorizer"></param>
+    /// <param name="exceptionInspector"></param>
+    /// <param name="factoryLoader"></param>
+    /// <param name="interceptors"></param>
+    public DataPortal(
+      ApplicationContext applicationContext, 
+      IDashboard dashboard, 
+      DataPortalServerOptions options,
+      IAuthorizeDataPortal authorizer,
+      List<IInterceptDataPortal> interceptors,
+      IObjectFactoryLoader factoryLoader,
+      IDataPortalActivator activator,
+      IDataPortalExceptionInspector exceptionInspector)
     {
       ApplicationContext = applicationContext;
       Dashboard = dashboard;
```

Comment:
Did you intend to be setting what is currently a static property from a constructor, or did you intend this to change to an instance property? The static declaration is shown above - it has changed, but only in its use of namespacing.